src/foam/core/Property.js

```diff
@@ -205,6 +205,36 @@ foam.CLASS({
           return comparePropertyValues(f(o1), f(o2));
         };
       }
+    },
+    {
+      name: 'diffPropertyValues',
+      transient: true,
+      factory: function() {
```

Comment:
Clarification from offline chat: @adamvy proposes that _the whole diff API_ live in a refinement package.